.github/workflows/commit-message.yml

```diff
@@ -9,6 +9,10 @@ on:
   push:
     branches:
       - main
+  # The "workflow_call" directive allows this workflow to be reused by other workflows.
+  # Example usage in the Slicer Custom Application Template:
+  # https://github.com/KitwareMedical/SlicerCustomAppTemplate/blob/main/.github/workflows/commit-message.yml
```

Comment:
Note that this URL will be valid only after integrating https://github.com/KitwareMedical/SlicerCustomAppTemplate/pull/62

Reply:
Also, as describe in https://github.com/Slicer/Slicer/pull/7898#issuecomment-2297525352, we were able to successfully reuse the workflow from the branch associated with `jcfr/Slicer` fork.